src/movies/moviesHandlers.ts

```diff
@@ -47,46 +30,9 @@ export class MovieController {
     }
   }
 
-  private async getMovieDetails(id: String) {
-    const url = `${tmdbUrl}${tmdbEndpoint}/${id}?api_key=${tmdbApiKey}&language=en-US&external_source=imdb_id`
-    try {
-      const resp = await needle('get', url)
-      return resp.body
-    } catch (error) {
-      console.log(error.message);
-    }
-  }
-
   private getImdbIDfromImdbURL(url: String) {
     const imdbID = url.substring(27, 36)
     return imdbID
   }
 
-  private constructMovie(tmdbMovieDetails: Movieresult, imdbURL: string) {
-    const newMovie = new movie();
-    try {
-      newMovie.original_title = tmdbMovieDetails.title
-      newMovie.image_url = `${tmdbThumbnailURL}${tmdbMovieDetails.poster_path}`
-      newMovie.is_highlight = false
-      newMovie.imdb_id = imdbURL
-      newMovie.release_date = tmdbMovieDetails.release_date
-      return newMovie
-    } catch (error) {
-      console.log(error)
```

Comment:
nur loggen ist nicht so geil, aber i.O. wegen mir. 

Frage: Was ist das Ziel von dem Code? Sollte dieser Teil vielleicht sogar einen Error werfen? Also einfach ohne try und catch. Damit das aufrufende Object feststellt: AH ich konnte kein neuen Movie erstellen! Ich muss was machen.

Reply:
Ziel: Quick and dirty, hauptsache es geht. Error management kommt als nächstes.